radiotools/helper.py

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, division, print_function  # , unicode_literals
+from radiotools.atmosphere import models as atm
```

Comment:
does this import trigger the calculation of the tables? If yes, please move it to the function where you need it.

Reply:
No it should not.